lua/telescope/actions.lua

```diff
@@ -34,6 +34,21 @@ function actions.get_selected_entry(prompt_bufnr)
   return actions.get_current_picker(prompt_bufnr):get_selection()
 end
 
+local function preview_scrolling(prompt_bufnr, termcode)
```

Comment:
Is this function used anywhere?

Reply:
Ahh that's some redundant function left out. Delete it:)